internal/controller/api/api.spec.json

```diff
@@ -64,6 +64,48 @@
         }
       }
     },
+    "/connection_status": {
+      "post": {
+        "tags": [
+          "api"
+        ],
+        "summary": "Retrieve the status of the connection.  This method will take the permitted tenants into consideration when looking up the connection details.",
+        "security": [
+          {
+            "ApiKeyAuth": []
+          },
+          {
+            "PSKAuthClientID": [],
+            "PSKAuthAccount": [],
+            "PSKAuthKey": []
+          }
+        ],
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/ConnectionStatusRequest"
```

Comment:
@dehort FWIW If you want to introduce this new resource as accepting org_id instead of account that would be OK for playbook-dispatcher. If you want to do it this way and introduce the org_id version in v2 later that's fine too.

Reply:
@jharting  Do you think it would be helpful to just go ahead and add org_id as an optional parameter to the /connection_status and /message operations?  I could go ahead and push this PR out and then add those optional parameters in a different PR.  I could certainly do that if we think that it makes the usage of the API a bit more clear.  What do you think?

Reply:
@jharting I think I'm really leaning towards adding the org_id to the existing /message operation and the new /connection_status operation. I think it makes the usage, logic, etc more clear.

Reply:
@dehort  Sounds good to me. With this new resource you could even leave out "account" entirely and only require org_id if you wanted (PD can handle that and there is nobody else to break as it's new). If you want to leave that in for consistency and only do these changes for all resources in v2 at some later point that's fine with me also.